reframe/frontend/cli.py

```diff
@@ -375,46 +375,27 @@ def main():
         # Locate and load checks
         try:
             checks_found = loader.load_all()
-        except OSError as e:
+        except OSError as oe:
```

Comment:
Why did you change to `oe`. If the change is valid the next line should raise from `oe`.

Reply:
The `e` should remain, unless its used elsewhere. This is a convention throughout the code. All exception objects are named `e`.

Reply:
Sorry, that is just a typo.